todos_repository_core/lib/src/todo_entity.dart

```diff
@@ -40,10 +48,24 @@ class TodoEntity {
 
   static TodoEntity fromJson(Map<String, Object> json) {
     return TodoEntity(
-      json['task'] as String,
-      json['id'] as String,
-      json['note'] as String,
-      json['complete'] as bool,
+      complete: json['complete'] as bool,
+      id: json['id'] as String,
+      note: json['note'] as String,
+      task: json['task'] as String,
+    );
+  }
+
+  TodoEntity copyWith({
```

Comment:
Please use the https://pub.dev/packages/copy_with package

Reply:
This package is a bit outdated and is giving me errors with other dependencies